mne/inverse_sparse/mxne_optim.py

```diff
@@ -768,6 +768,115 @@ def norm_l1_tf(Z, shape, n_orient):
     return l1_norm
 
 
+def norm_epsilon(Y, l2_ratio):
+    """Dual norm of l2_ratio * L2 norm + (1. - l2_ratio) * L1 norm, at Y.
+
+    This is the unique solution in nu of
+    norm(prox_l1(Y, nu * (1 - l2_ratio)), ord=2) = l2_ratio * nu
+
+    Parameters
+    ----------
+    Y : array, shape (n_atoms,)
+        The input data.
+    l2_ratio : float between 0 and 1
+        Tradeoff between L2 and L1 regularization.
```

Comment:
in sklearn we prefer to parametrize with l1_ratio

Reply:
I've changed to l1_ratio